### PR TITLE
Generate index.json on docker image build stage

### PR DESCRIPTION
**Pull request by skabashnyuk**
### What does this PR do?
Generate index.json on docker image build stage

Reference to https://github.com/eclipse/che/issues/10745

### CODE REVIEW

File: index.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+## search for all editors and plugins
+declare -a arr=(`find . -name "meta.yaml"`)
+FIRST_LINE=true
+echo "["
+## now loop through meta files
+for i in "${arr[@]}"
+do
+    IFS='/' segments=($i)
```
Comment (garagatyi) [inline index.sh]: Can you elaborate what this command does?

Comment (skabashnyuk) [reply index.sh]: splitting path variable on segments

**Comment by garagatyi** [comment]
@skabashnyuk Would you be willing to add in Readme how to get metafiles and index? I mean an example for a client with the index URL and how to construct URLs for meta files listed in the index.

**Comment by skabashnyuk** [comment]
>@skabashnyuk Would you be willing to add in Readme how to get metafiles and index? I mean an example for a client with the index URL and how to construct URLs for meta files listed in the inde

ok

**Pull request merged**